types/react-big-calendar/react-big-calendar-tests.tsx

```diff
@@ -143,10 +145,7 @@ class CalendarResource {
 
 // overriding 'views' props
 {
-    interface DayProps {
-        random: string;
-    }
-    class DayComponent extends React.Component<DayProps> {
+    class DayComponent extends React.Component {
```

Comment:
Why does this no longer work? If this is an expected type error we should rather explicitly mark it with `$ExpectError` and explain why.

Reply:
`DayComponent`s prop `random` is defined as a mandatory prop, but when `react-big-calendar` uses this component as a view, this prop would never be assigned. The test tests if components can be assigned as views, so it is ok, but maybe it's worth to add another $ExpectError test? What do you think?

Reply:
I would add it as a expected failure test. Never hurts to add these.